### ISSUE TITLE
How to run multiple newsboat on different machines, sharing database?

### ISSUE THREAD

**Issue by uzsolt** [newsboat/newsboat#2047]
I'm using newsboat many years. Now would be nice if can share database between machines.

The situation: I've two machines, one at home and one at work. I want use newsboat on both - but if I read an article on HostA, I don't want read this on HostB, so should share cache.db. But how?

Ideas:

1. Upload cache.db to "cloud" if modified, and download first on the other host. I think it's an ugly workaround.
2. Use a newsboat on a VPS but it's difficult to open URL on localhost (see https://github.com/newsboat/newsboat/issues/2034).
3. Mount a remote directory with cache.db. But if loose the network newsboat will exit (or create a new cache.db).
4. Any other?

Maybe the best solution would be support MySQL/MariaDB or similar database server in newsboat. What do you think? Is it possible?

**Comment by Minoru** [comment]
The most common solution is to use one of [the supported newsreading services](https://newsboat.org/releases/2.27/docs/newsboat.html#_newsboat_as_a_client_for_newsreading_services). Would that work for you?

I don't think it'd be easy to adapt Newsboat to something other than SQLite, because Newsboat is designed with an assumption that it "owns" the database. Adding support for concurrent reads and writes would take some work.

We also have some other suggestions and ideas, like #381 and #471, but none of them are fleshed out yet.

**Comment by uzsolt** [comment]
I tried FreshRSS but I got authentication error (maybe my fault). But as I saw newsbeuter created _cache.db_. I thought newsboat stores the articles status (unread/read) in own database and doesn't sync it with the FreshRSS.

But now I read it in https://github.com/newsboat/newsboat/issues/381#issuecomment-468956949:
"Every time an article or feed is marked as (un)read, Newsboat sends this to the syncing service."

So if it's right I think it's a good solution for me. I'll test it. Thanks!
